cli/commands/output-module-groups/command.go

```diff
@@ -6,13 +6,42 @@ import (
 )
 
 const (
-	CommandName = "output-module-groups"
+	CommandName       = "output-module-groups"
+	SubCommandApply   = "apply"
+	SubCommandDestroy = "destroy"
 )
 
 func NewCommand(opts *options.TerragruntOptions) *cli.Command {
 	return &cli.Command{
-		Name:   CommandName,
-		Usage:  "Output groups of modules ordered for apply as a list of list in JSON (useful for CI use cases).",
+		Name:        CommandName,
+		Usage:       "Output groups of modules ordered by command (apply or destroy) as a list of list in JSON (useful for CI use cases).",
+		Description: "The command will recursively find terragrunt modules in the current directory tree and output the list of list in JSON for the terraform command in dependency order (unless the command is destroy, in which case the command is run in reverse dependency order).",
+		Subcommands: subCommands(opts),
+		Action:      func(ctx *cli.Context) error { return Run(opts.OptionsFromContext(ctx)) },
+	}
+}
+
+func subCommands(opts *options.TerragruntOptions) cli.Commands {
+	cmds := cli.Commands{
+		applySubCommandFunc(opts),
+		destroySubCommandFunc(opts),
+	}
+
+	return cmds
+}
+
+func applySubCommandFunc(opts *options.TerragruntOptions) *cli.Command {
+	return subCommandFunc(SubCommandApply, opts)
+}
+
+func destroySubCommandFunc(opts *options.TerragruntOptions) *cli.Command {
+	return subCommandFunc(SubCommandDestroy, opts)
+}
+
+func subCommandFunc(cmd string, opts *options.TerragruntOptions) *cli.Command {
+	opts.TerraformCommand = cmd
+	return &cli.Command{
+		Name:   cmd,
```

Comment:
will be helpful to define `Usage` to show how subcommands can be used

![image](https://github.com/gruntwork-io/terragrunt/assets/10694338/e1378b23-5391-4bf7-98f1-2241e2546ce1)

Reply:
I originally had usage for the sub-commands but felt it was repetitive. updated!